packages/core/src/components/tabs/tabTitle.tsx

```diff
@@ -71,7 +71,7 @@ export class TabTitle extends AbstractPureComponent2<TabTitleProps> {
                 id={generateTabTitleId(parentId, id)}
                 onClick={disabled ? undefined : this.handleClick}
                 role="tab"
-                tabIndex={disabled ? undefined : selected ? 0 : -1}
+                tabIndex={disabled ? undefined : selected ? -1 : 0}
```

Comment:
do you have any documentation to justify this change?

Reply:
If the tab is _not_ selected, you'd want to be able to use the tab button to select it, hence setting `tabIndex=0`.

If the tab _is_ currently selected, clicking does nothing, so `tabIndex=-1` since `onClick` is also undefined.